pkg/kubelet/cm/cpumanager/cpu_assignment_test.go

```diff
@@ -214,7 +218,8 @@ func TestCPUAccumulatorFreeNUMANodes(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.description, func(t *testing.T) {
-			acc := newCPUAccumulator(tc.topo, tc.availableCPUs, 0, CPUSortingStrategyPacked)
+			logger, _ := ktesting.NewTestContext(t)
```

Comment:
I think `t` is the same here as at the start of `TestCPUAccumulatorFreeNUMANodes`, so it should be enough to call `ktesting.NewTestContext(t)` there once.

Reply:
good point, fixed and fixed all the similar instances